src/app/application-form/application-form.component.html

```diff
@@ -0,0 +1,573 @@
+                  <div *ngIf="totalAmount.errors['min']">
+                    Amount should at least be 10000
+                  </div>
+                </mat-error>
+              </mat-form-field>
+
+              <div class="down_payment_field">
+                <mat-form-field
+                  appearance="fill"
+                  [floatLabel]="getLoanFormFloatLabelValue()"
+                >
+                  <mat-label for="downPayment">Down payment</mat-label>
+                  <input
+                    matInput
+                    type="number"
+                    formControlName="downPayment"
+                    id="downPayment"
+                    placeholder="Down payment"
+                    [errorStateMatcher]="matcher"
+                  />
+                  <mat-error *ngIf="downPayment.errors && downPayment.touched">
+                    <div *ngIf="downPayment.errors['required']">
+                      Down payment is required
+                    </div>
+                    <div *ngIf="downPayment.errors['pattern']">
+                      Only numeric values allowed
+                    </div>
+                  </mat-error>
+                  <mat-error
+                    *ngIf="loanForm?.errors?.['downPaymentTooSmall'] && downPayment.touched"
+                  >
+                    <div>Down payment should be at least 15 %</div>
+                  </mat-error>
+                </mat-form-field>
+                <div class="percentage">{{ percentage }} %</div>
+              </div>
+
+              <mat-form-field
+                appearance="fill"
+                [floatLabel]="getLoanFormFloatLabelValue()"
+              >
+                <mat-label for="loanTerm">Loan term</mat-label>
+                <input
+                  matInput
+                  type="number"
+                  formControlName="loanTerm"
+                  id="loanTerm"
+                  placeholder="Loan term"
+                />
+                <mat-error *ngIf="loanTerm.errors && loanTerm.touched">
+                  <div *ngIf="loanTerm.errors['required']">
+                    Loan term is required
+                  </div>
+                  <div *ngIf="loanTerm.errors['pattern']">
+                    Only numeric values allowed
+                  </div>
+                  <div *ngIf="loanTerm.errors['min']">
+                    Loan term should be at least 1 year
+                  </div>
+                  <div *ngIf="loanTerm.errors['max']">
+                    Loan term cannot exceed 30 years
+                  </div>
+                </mat-error>
+              </mat-form-field>
+              <div>
+                <button mat-button matStepperPrevious>Back</button>
+                <button mat-button matStepperNext type="button">Next</button>
+              </div>
+            </div>
+          </mat-step>
+          <!-- ------------------------------------------------------------------------------------- -->
```

Comment:
If you are using this for different parts, try and make the separator the same length as others and leave a blank line before.